src/main/java/com/rsmart/kuali/kfs/cr/batch/GlTransactionStep.java

```diff
@@ -297,11 +297,15 @@ public boolean execute(String jobName, Date jobRunDate) throws InterruptedExcept
     }
     
     private void logClassLoaderDebugInfo() {
+        Class classHelperClass = org.apache.ojb.broker.util.ClassHelper.class;
+        LOG.info("logClassLoaderDebugInfo:: OJB Broker ClassHelper.ClassLoader.name: {}", 
+                (ObjectUtils.isNull(classHelperClass) || ObjectUtils.isNull(classHelperClass.getClassLoader()) ? "ClassHelper or ClassLoader IS NULL" : classHelperClass.getClassLoader().getName()));

```

Comment:
Just to clarify, the class loader that should be logged here (and in the similar methods) is the one from `ClassHelper.getClassLoader()`, not the one from `ClassHelper.class.getClassLoader()`.

Also, for consistency with the other logged class loaders, I would suggest logging its `toString()` value (implicitly or explicitly) instead of logging the `getName()` result.